consumers/messages.go

```diff
@@ -59,7 +59,10 @@ func handle(t transformers.Transformer, c Consumer) handleFunc {
 				return err
 			}
 		}
-		return c.Consume(m)
+		errs := make(chan error)
+		c.Consume(m, errs)
```

Comment:
This will not work. You need to call the correct method - i.e. you need an indicator of which mode you want to use.

Reply:
Can I add a `async bool` variable to the consumer interface, which returns which mode is being used? @dborovcanin

Reply:
No need to change the interface, but you can change `Start` method of `conusmers/messages.go` to accept this bool value.